### PR TITLE
SISRP-41853 - Updates requested after testing, changes ratio to percentage

### PR DESCRIPTION
**Pull request by davidlimcheng**
https://jira.berkeley.edu/browse/SISRP-41853

* Updates in the JIRA, see attachment "2019-02-13 Design Updates"

### CODE REVIEW

File: src/assets/templates/widgets/academics/degree_progress_undergrad.html [hunk 1]
```diff
@@ -79,7 +79,7 @@ <h2>Degree Progress</h2>
     </div>
     <div data-ng-if="!degreeProgress.undergraduate.showPnpCalculator">
       </br>
-      <button data-ng-click="showPnpCalculator()" class="cc-button-link" data-ng-disabled="isAdvisingStudentLookup">Passed (P) Grade Limit Estimator</button>
+      <button data-ng-click="showPnpCalculator()" class="cc-button-link" data-ng-disabled="isAdvisingStudentLookup">1/3 Passed (P) Grade Limit Estimator</button>
```
Comment (nkutub) [inline src/assets/templates/widgets/academics/degree_progress_undergrad.html]: Is this intentional? @davidlimcheng

File: src/react/components/pnp_calculator/PassedGradeLimitEstimator.js [hunk 2]
```diff
@@ -30,23 +30,32 @@ const renderTransferUnitNote = (hasTransferUnitInput) => {
   if (hasTransferUnitInput) {
     return (
       <p className="cc-react-text--small cc-react--no-margin">
-        Note: Only exams completed as a high school student are eligible for test credit. Lower division transfer credit may not exceed 70 semester units.
+        *Note: Transfer credit includes both course and exam credit. Only exams completed as a high school student are eligible for exam credit. Loser division
```
Comment (nkutub) [inline src/react/components/pnp_calculator/PassedGradeLimitEstimator.js]: loser or lower @davidlimcheng

File: src/assets/templates/widgets/academics/degree_progress_undergrad.html (hunk 1 above)
Comment (davidlimcheng) [reply src/assets/templates/widgets/academics/degree_progress_undergrad.html]: @nkutub Yep as designed

File: src/react/components/pnp_calculator/PassedGradeLimitEstimator.js (hunk 2 above)
Comment (davidlimcheng) [reply src/react/components/pnp_calculator/PassedGradeLimitEstimator.js]: lol, will amend!

**Comment by davidlimcheng** [comment]
Amended, thx @nkutub

**Pull request merged**